_examples/profile/.gitignore

```diff
@@ -3,3 +3,6 @@
 # Example project generated self-signed certificate
 /yotiSelfSignedCert.pem
 /yotiSelfSignedKey.pem
+
+# Compiled binary
+/profile
```

Comment:
New line

Reply:
Added in https://github.com/getyoti/yoti-go-sdk/pull/226/commits/a072aadfae3cd4d62a949e0ae7bfee63c9b06960